inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java

```diff
@@ -1196,8 +1197,8 @@ private List<InlongClusterNodeEntity> getClusterNodes(String groupId, String clu
         InlongGroupEntity groupEntity = groupMapper.selectByGroupId(groupId);
         if (groupEntity == null) {
             String msg = "inlong group not exists for groupId=" + groupId;
-            LOGGER.debug(msg);
-            throw new BusinessException(msg);
+            LOGGER.info(msg);
```

Comment:
For InlongGroup not found, the request parameters seem unreasonable, so is it more reasonable to use warn-level logs?

Reply:
done